package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "now": {
     "env": [
-      "GH_TOKEN"
+      "GH_TOKEN",
+      "CORS"
```

Comment:
It looks like this isn't the environment variable used below. Should this be changed to HOST_NAME?